### DISCUSSION TITLE
leetro mpc compatibility

### DISCUSSION THREAD

**Discussion by Badrimus** [meerk40t/meerk40t#2464] [General]
hi, 
can meerk40t control leetro controller? 
thinking of buying a machine that comes with this controller but lasercut 5.3 fromm what i´ve seen not very solid software and lucks some nice and useful features that meerk40t has

**Comment by tatarize** [comment]
You should be able to buy an M3 Nano Plus for like 35 bucks. Or a fairly cheap GRBL controller. I've not gotten any good direct access to a leetro controller and probably would need one to do the reverse engineering.

**Reply by Badrimus** [reply]
![Screenshot_20240216-113615~2](https://github.com/meerk40t/meerk40t/assets/155115897/c7212559-718a-4fb2-b011-7b02babd5914)

This is the controller, the machine is big and it has 100w tube 90cm X 60cm bed size, do you think that M3 nano is a better option for this machine? Why these controllers (ruida also) are more expensive in comparison with M3 nano and mks dlc?

**Reply by Sophist-UK** [reply]
Here is the spec / logical connection diagram for the MPC6565: http://dan.spb.ru/soft/Leetro/manuals/MPC6565.pdf
Some sort of manual here: https://www.scribd.com/document/401078083/MPC6535-6565-DSP5-3-Software-Manual-en-V1-6
Some screen shots of the software here: http://www.destiny-ideas.com/b/6/

Whilst new MPC6565 are a c. $300 on Alibaba, a bit of research says that they are not worth much 2nd hand (probably because the control codes are proprietary and the only software that supports it is not very good).

But hopefully with the diagram for the MPC6565 and a similar one for the m2/3 nano, you should be able to work out how to connect the wires to the m2/3 nano when you remove them from the MPC6565.

**Reply by Badrimus** [reply]
Very useful, thanks a mil 🙂
